### ISSUE TITLE
duplicate property in JSON

### ISSUE THREAD

**Issue by pduchesne** [interactive-instruments/ldproxy#96]
some features have duplicate properties, which causes some JSON parsers to fail (in particular the one embedded in swagger.js).

For instance : 
http://wfs3hackathon.ldproxy.net/rest/services/test/ProtectedSite/_1_3_NL9802060_/?f=json

see the duplicate `siteDesignation.percentageUnderDesignation`property.

JSON as a spec does not prevent it explicitly, but most parsers choke on it.

**Comment by azahnen** [comment]
Yeah, I think the best option in this case would be to deactivate the property. I will do that.

If you look at http://wfs3hackathon.ldproxy.net/rest/services/test/ProtectedSite/_1_3_NL9802060_/?f=xml you see that one of the properties is for designation specialAreaOfConservation and one for specialProtectionArea. Without the designations, the property is not really useful.

But due to some open issues like #4 and #50, it is currently not possible to map this in a useful way.

**Comment by azahnen** [comment]
I implemented a very simple solution for #4, so that the generated JSON is at least valid, and reactivated the property.
```
"siteDesignation.percentageUnderDesignation" : 100,
"siteDesignation.percentageUnderDesignation.2" : 100,
```
